src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -49,8 +51,10 @@ public CommandResult execute() {
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            List<String> wordsInName = person.getName().getWordsInName();
+            wordsInName.replaceAll(String::toLowerCase);
+            final Set<String> editedWordsInName = new HashSet<>(wordsInName);
```

Comment:
lowercasedWordsInName would be more descriptive than edited. Just a minor comment.